google/cloud/storage/examples/run_examples_utils.sh

```diff
@@ -262,40 +268,53 @@ run_all_object_examples() {
 ################################################
 run_all_object_rewrite_examples() {
   local source_bucket_name=$1
-  local destination_bucket_name=$2
+  local target_bucket_name=$2
   shift 2
 
-  local object_name="rewrite-source-object-$(date +%s)-${RANDOM}.txt"
-  local rewrite_object_name="rewrite-object-$(date +%s)-${RANDOM}.txt"
+  local source_object_name="rewrite-source-object-$(date +%s)-${RANDOM}.txt"
```

Comment:
Nit: reusing variable names and could potentially be confusing later. Either rename variables to reflect case or add a comment restating reuse.

Reply:
Split across several functions to fix this.